nemo/core/classes/common.py

```diff
@@ -247,8 +247,7 @@ def from_config_dict(cls, config: DictConfig, *args: Any, **kwargs: Any):
             instance = hydra.utils.instantiate(config=config, *args, **kwargs)
         else:
             # models are handled differently for now
-            # TODO: allow passthrough for args, and kwargs too?
-            instance = cls(cfg=config)
+            instance = cls(cfg=config, *args, **kwargs)
```

Comment:
please revert this change for now

Reply:
This change is helping me restoring model else I am getting garbage values

Reply:
Can you inspect what are the values of *args and **kwargs. What special values are going into your `cls` that with them it gets proper embeddings but without it results in garbage values.